docs/website-settings.md

```diff
@@ -28,6 +29,35 @@ Here you can set up a whitelist by defining a whitelist name or its ID from [whi
 
 ![set-whitelist](/img/content/set-whitelist.png)
 
+## Configurting player restrictions
+
+Since not all regions of the world participate in the ETF2L competitions nor have a profile created there, this option allows you to:
+
+- enable/disable user validation against the [ETF2L API](https://api.etf2l.org),
+- define minimum TF2 in-game time spent in order to register to the website.
+
+The ETF2L user validation lets website:
+
+- assign an ETF2L profile to the tf2pickup.org profile,
+- block player registration if the player is blacklisted/banned from the ETF2L,
+- assign the nickname to a player (without it, the current nickname on steam is assigned to a tf2pickup.org profile).
```

Comment:
For all bullet point sentences, 

1) Start with a capital letter
2) End with a period

Reply:
@garrappachc do you agree with this? I don't.

Reply:
Usually, I'll hear to start bullet point with capital letters, but sometimes ends with periods. The periods are more for consitancy than anything- omitting them here means to omit them everywhere though (😬)

Reply:
I have no idea what the bullet points work like in English, I simply assumed they work the same way they do in Polish. In Polish, you can either use "sentence equivalent" and then treat each point separately, or you can list things as if they were a part of a regular sentence - then you use commas to separate them.

Reply:
I've looked on some random repositories and I've changed my mind, because that's a thing. I'll change it soon.

Reply:
😎 English is Awesome 😎